### PR TITLE
comment out report

### PR DESCRIPTION
**Pull request by yasun1**

### CODE REVIEW

**Comment by yasun1** [comment]
/pj-rehearse periodic-ci-openshift-openshift-tests-private-master-rosacli-aws-rosa-hcp-byo-kms-oidc-auditlog-e2e-f3

File: ci-operator/step-registry/rosa/test/e2e/rosa-test-e2e-commands.sh [hunk 1]
```diff
@@ -61,10 +59,9 @@ junit_xml="${TEST_PROFILE}-$(date +%m%d%s).xml"
 rosatest --ginkgo.v --ginkgo.no-color \
   --ginkgo.timeout ${TEST_TIMEOUT} \
   --ginkgo.junit-report "${ARTIFACT_DIR}/$junit_xml" \
-  ${LABEL_FILTER_SWITCH} \
-  | sed "s/$AWS_ACCOUNT_ID/$AWS_ACCOUNT_ID_MASK/g" || true
```
Comment (radtriste) [inline ci-operator/step-registry/rosa/test/e2e/rosa-test-e2e-commands.sh:65]: why did you remove the Mask ?

Comment (yasun1) [reply ci-operator/step-registry/rosa/test/e2e/rosa-test-e2e-commands.sh:65]: Now it is been masked here: https://github.com/openshift/rosa/blob/master/tests/utils/log/constants.go#L23-L24

**Review by radtriste — APPROVED** [review]
/lgtm
